File: Calendar/app/src/main/java/com/example/calendar/MainActivity.java
package com.example.calendar;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.CalendarView;
import android.widget.GridView;
import android.widget.LinearLayout;
import android.widget.TextView;

import java.text.SimpleDateFormat;
import java.util.Date;

public class MainActivity extends AppCompatActivity {

    private TextView titleText;
    private Button prevButton, nextButton;
    private CalendarAdapter mCalendarAdapter;
    private GridView calendarGridView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button button= findViewById(R.id.button);
        calendarGridView calendar=
        String timeStamp2 = new SimpleDateFormat("yyyyMMdd").format(new Date());
        button.setOnClickListener(v->{
            calender

        });
    }
}

File: SampleApp2/app/src/main/java/com/example/sampleapp2/MainActivity.kt

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.TextView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() , View.OnClickListener{

    // タップ数を格納しておくための変数
    var count: Int = 0
    // TextViewのインスタンスを格納する変数
    var countText: TextView? = null


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // idからTextViewのオブジェクトを取得
        countText = this.findViewById(R.id.count)
        countText!!.text = count.toString()

        // idからButtonのオブジェクトを取得
        val btn: Button = this.findViewById(R.id.btn)
        // ボタンがタップを検知するように
        btn.setOnClickListener{
            count += 1
            countText!!.text = count.toString()
        }
    }

}